src/lib/FlightTasks/FlightTasks.cpp

```diff
@@ -47,6 +47,23 @@ const vehicle_constraints_s FlightTasks::getConstraints()
 	}
 }
 
+const vehicle_trajectory_waypoint_s FlightTasks::getAvoidanceWaypoint()
```

Comment:
What confuses me most is the naming avoidance_waypoint even though the type is trajectory_waypoint.

I guess the intention is in the future the trajectory_waypoint should become the general setpoint for "curved"/smooth flying but right now the type is only used to inform and get commands from the avoidance system. Right?

Reply:
exactly